Package.swift

```diff
@@ -50,6 +50,10 @@ let package = Package(
         "PythonKit",
         "CTensorFlow",
         .product(name: "Numerics", package: "swift-numerics"),
+      ],
+      swiftSettings: [
+        .define("USING_X10_BACKEND"),
```

Comment:
From offline discussion: it seems like we can remove the `USING_X10_BACKEND` flag and checks soon, since that is the only path forward for enabling SwiftPM builds for `tensorflow/swift-apis`.